.github/workflows/dockerimage.yml

```
@@ -1,5 +1,8 @@
 name: Docker Compose Actions Workflow
 
+on:
+  workflow_dispatch:
```

I believe, according to the documentation, if you don't pass any input parameters, you may do it on a single line, which in addition seems to me a bit more readable:

```suggestion
on: workflow_dispatch
```

Docs: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example

---

nice! thanks